Unit1/Module1.6/Mini-Project1.6/README.md

```diff
@@ -0,0 +1,12 @@
+# Mini Project 1.6: Deploy your App on Heroku
+
+##### Credit to OpenClassrooms
+![Become](https://github.com/OCclassprojects/logo/blob/master/fav-icon.png?raw=true)
+
+### Mini Project Instructions
+
+1. Open the [project page on OpenClassrooms](https://openclassrooms.com/en/courses/4614791-deploy-apps-on-heroku)
+
+2. Submit the link to your Heroku app in the form below. 
```

Comment:
Since we arent using the turn in Form at this point (that will only be used for Unit 1 of Fundamentals), we should instruct them to put the link in a text file and submit as a PR.